packages/react-components/lib/reports/log-management/search-log-form.tsx

```diff
@@ -27,39 +51,31 @@ export const SearchLogForm = (props: SearchLogFormProps): React.ReactElement =>
   // The log contains information from different services, the label help us differentiate the service
   const [logLabel, setLogLabel] = React.useState('');
   const [logLevel, setLogLevel] = React.useState(LogLevel.All);
-  const [fromLogDate, setFromLogDate] = React.useState<MaterialUiPickersDate>(new Date());
-  const [toLogDate, setToLogDate] = React.useState<MaterialUiPickersDate>(new Date());
-
-  const classes = useStyles();
+  const [fromLogDate, setFromLogDate] = React.useState<Date>(new Date());
+  const [toLogDate, setToLogDate] = React.useState<Date>(new Date());
 
   const searchQuery = () => {
     search && search({ toLogDate, fromLogDate, logLabel, logLevel });
   };
 
-  const handleLogLabelChange = React.useCallback(
-    (event: React.ChangeEvent<{ name?: string; value: unknown }>) => {
-      setLogLabel(event.target.value as string);
-    },
-    [],
-  );
+  const handleLogLabelChange = React.useCallback((event: SelectChangeEvent<React.ReactText>) => {
+    setLogLabel(event.target.value as string);
+  }, []);
 
-  const handleLogLevelChange = React.useCallback(
-    (event: React.ChangeEvent<{ name?: string; value: unknown }>) => {
-      setLogLevel(event.target.value as LogLevel);
-    },
-    [],
-  );
+  const handleLogLevelChange = React.useCallback((event: SelectChangeEvent<React.ReactText>) => {
+    setLogLevel(event.target.value as LogLevel);
+  }, []);
 
-  const handleFromLogDateChange = React.useCallback((date: MaterialUiPickersDate) => {
+  const handleFromLogDateChange = React.useCallback((date: any) => {
```

Comment:
Don't use `any`, you can use `Required<DateAndTimePickers>['onChange']` to grab the callback type, or use `Date` since we already know `fromLogDate` is a `Date`.